Library/Formula/mdbm.rb

```diff
@@ -0,0 +1,23 @@
+class Mdbm < Formula
+  desc "super-fast memory-mapped key/value store."
+  homepage "http://yahooeng.tumblr.com/post/104861108931/mdbm-high-speed-database"
+  url "https://github.com/yahoo/mdbm/archive/v4.12.3.tar.gz"
+  sha256 "1bdd27696980b8234893f2c7bfbd0d1ad5c06ccdb1eb91bcf053db27c61eea26"
+
+  depends_on "coreutils"
+  depends_on "cppunit"
+  depends_on "readline"
+  depends_on "openssl"
+
+  def install
+    ENV.delete "CC"
```

Comment:
Clang appears to have different behavior (different default warnings) when called as `clang++` rather than `c++`, which causes the build to fail due to `-Werror`. I assume it's selecting a different C++ standard depending on the argv[0] it's called by.

This appears to be a new warning which did not exist in whatever xcode version we used in 2015 to do the osx port.

The actual error is a safe cast between size_t and int performed in a unit test. I'll push a fix for it upstream when I get a chance, but it's harmless and should be ignored for now.